contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-python.mdx

```diff
@@ -58,6 +58,20 @@ posthog.capture(
 )
 ```
 
+
+### Sending `$feature_flag_called` events
+
+By default, we send a `$feature_flag_called` event whenever you call `posthog.get_feature_flag()` or `posthog.is_feature_enabled()` and it's a new user, or the value of the flag has changed. This is useful for analytics, and knowing when a flag was accessed for a user.
+
+> *Note:* Tracking whether it's a new user, or whether the flag value is changed happens in a local cache, which means if you reinitialise the PostHog client, this cache will reset as well, and can end up sending events again for old users as well. This is not a problem. 
```

Comment:
What do we mean by "This is not a problem"? What happens if we send events again?

Reply:
Fair question!

```suggestion
> *Note:* Tracking whether it's a new user, or whether the flag value is changed happens in a local cache, which means if you reinitialise the PostHog client, this cache will reset as well, and can end up sending events again for old users as well. This is not a problem because we aggregate analytics by unique users so these duplicates don't affect things.
```